app/ui/src/app/core/providers/user-provider.service.ts

```diff
@@ -26,6 +35,6 @@ export class UserProviderService extends UserService {
    * Log the user out
    */
   logout(): Observable<any> {
-    return this.apiHttpService.setEndpointUrl('/oauth/sign_out').get();
+    return this.httpClient.get('/oauth/sign_out');
```

Comment:
This change probably isn't necessary.  If the GET request isn't being triggered we probably just need to subscribe to this Observable, which we'd want to do anyways to trigger a redirect...  TBH I wouldn't inject httpClient directly in this class but instead minimize the changes and add the subscription...

Reply:
this change is here to avoid the path prefix the apiHttpService adds to all endpoints.

Reply:
Ah of course, +1 then :-)